### PR TITLE
[Feature] added header

### PR DESCRIPTION
**Pull request by Arthur-Tolkachov**
## Description
- [DEMO LINK](https://arthur-tolkachov.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://arthur-tolkachov.github.io/layout_moyo-header/report/html_report/)

## Checklist
- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,12 +11,106 @@
...
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="/"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (volodymyr-soltys97) [inline src/index.html:55]: Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+
+  --nav-font-color: #000;
+  --primary-color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/style.css:7]: Create a variables with root selector

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
```
Comment (volodymyr-soltys97) [inline src/style.css:15]: This color is default
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,69 @@
...
+  align-items: center;
+  height: 60px;
+  font-weight: 500;
+  text-decoration: none;
+  color: var(--nav-font-color);
+  position: relative;
+  font-size: 0.75rem;
+}
+
+.nav__link::after {
```
Comment (volodymyr-soltys97) [inline src/style.css:45]:
```suggestion
.is-active::after {
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,69 @@
...
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--primary-color);
+  border-radius: 8px;
+  transform: scaleX(0);
+  transform-origin: left;
```
Comment (volodymyr-soltys97) [inline src/style.css:55]:
```suggestion
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,69 @@
...
+  color: var(--primary-color);
+}
+
+.nav__link.is-active {
+  color: var(--primary-color);
+}
+
+.nav__link.is-active::after {
+  transform: scaleX(1);
 }
```
Comment (volodymyr-soltys97) [inline src/style.css:69]:
```suggestion
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,69 @@
...
+  transform: scaleX(0);
+  transform-origin: left;
+  transition: transform 0.2s ease;
+}
+
+.nav__link:hover {
+  color: var(--primary-color);
+}
+
+.nav__link.is-active {
```
Comment (volodymyr-soltys97) [inline src/style.css:63]:
```suggestion
.is-active {
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

File: src/style.css (hunk 4 above)
Comment (Arthur-Tolkachov) [reply src/style.css:45]: If I do this, my transition won't work correctly, won't it?

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**